src/scripts/wildBattle/RouteData.ts

```diff
@@ -474,6 +474,8 @@ Routes.add(new RegionRoute(
     'Pinkan Forest', GameConstants.Region.kanto, 41,
     new RoutePokemon({
         land: ['Pinkan Caterpie', 'Pinkan Weedle', 'Pinkan Pidgey', 'Pinkan Vileplume', 'Pinkan Paras', 'Pinkan Venonat', 'Pinkan Mankey', 'Pinkan Exeggutor'],
+        special:
+    [new SpecialRoutePokemon(['Pinkan Pidgeotto'], new ObtainedPokemonRequirement(pokemonMap['Pinkan Pidgeotto']))],
```

Comment:
It does not show up in the forest, in the anime, which is why we did not add it to the routes.
For the rest of the mons, i will ask someone who have played more of the games to look it through

Reply:
This one was the only one I took the liberty of adding that is outside of normally obtainable areas.